packages/controls/src/widget_description.ts

```diff
@@ -65,14 +67,19 @@ class DescriptionView extends DOMWidgetView {
 
     updateDescription() {
         let description = this.model.get('description');
+        let tooltip = this.model.get('tooltip');
+        if (tooltip.length === 0) {
+            tooltip = description;
```

Comment:
How would we disable the tooltip if we didn't want it to show up?

Reply:
As far as I understand, it couldn't be disabled completely previously also since it always defaults to the description. Not specifying the tooltip or setting it to an empty string will result in closest behavior.